project/cmake/addons/addons/adsp.basic/adsp.basic.txt

```diff
@@ -1 +1 @@
-adsp.basic https://github.com/AchimTuran/adsp.basic master
+adsp.basic https://github.com/AchimTuran/adsp.basic fix_win32_cmake
```

Comment:
in my system i use a git clone, in this system they use the tarball download functionality from github.
it means that as long as you use a branch name and not a fixed hash, no updates will ever occur cause it will think the tarball is the same, even if the branch contents changed.

short version; yes, what you have is as correct as it gets using the current system.